### ISSUE TITLE
Allow parsing of Lucee Static Components

### ISSUE THREAD

**Issue by cybersonic** [cflint/CFLint#700]
Lucee allows the ability to have static functions in components, i.e. calling a function without having to instantiate the Component itself (see https://docs.lucee.org/guides/cookbooks/StaticScope.html ) 

The syntax breaks CFLint parsing and brings false positives in parsing:

```
<cfscript>
isValid = com.domain.services.Service::isUserValid(userid);
</cfscript>
```

This throws a:
`Critical: End of statement(;) expected after Service`

error

**Comment by KamasamaK** [comment]
This is also supported as of ColdFusion 2021, so no longer Lucee-exclusive.

https://helpx.adobe.com/coldfusion/developing-applications/building-blocks-of-coldfusion-applications/building-and-using-coldfusion-components/cfc-variables-and-scope.ug.html#static-support

**Comment by cybersonic** [comment]
Cool, so it is not just Lucee then. Great!
